lxd/instance/instancetype/instance.go

```diff
@@ -28,6 +28,12 @@ const (
 // ConfigVolatilePrefix indicates the prefix used for volatile config keys.
 const ConfigVolatilePrefix = "volatile."
 
+// ConfigKeyPrefixesAny indicates valid prefixes for configuration options.
+var ConfigKeyPrefixesAny = []string{"environment.", "user.", "image."}
```

Comment:
Should this slice also include `ConfigVolatilePrefix`?

Reply:
Sure!

Reply:
I don't think so - we want to leave handling of `volatile` keys to `ConfigKeyChecker`.